linkis-computation-governance/linkis-manager/linkis-application-manager/src/test/resources/application.properties

```diff
@@ -0,0 +1,39 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+
+#h2 database config
+spring.datasource.driver-class-name=org.h2.Driver
+#init
+spring.datasource.url=jdbc:h2:mem:test;MODE=MySQL;DB_CLOSE_DELAY=-1;DATABASE_TO_LOWER=true
+spring.datasource.username=sa
+spring.datasource.password=
+spring.datasource.schema=classpath:create.sql
+
+#pgtest
+#spring.datasource.driver-class-name=org.postgresql.Driver
+#spring.datasource.url=jdbc:postgresql://${host}:${port}/linkis
+#spring.datasource.username=linkis
+#spring.datasource.password=linkis
+
+mybatis-plus.mapper-locations=classpath*:mapper/common/mysql/*.xml
```

Comment:
mapper/common, is for general,  the mysql should move to path  mapper/mysql,  and pg to mapper/postgresql.
almost all database support sql92.  so unless the sql used a database specific function,  others can be shared.

the mapper-locations can have more the one,  classpath*:/mapper/common/*.xml,  classpath*:/mapper/mysql/*.xml,

Reply:
Thank you for your suggestion. I have modified it according to your suggestion.